flood_adapt/object_model/io/unitfulvalue.py

```diff
@@ -49,7 +49,15 @@ class VerticalReference(str, Enum):
     datum = "datum"
 
 
-class UnitfulLength(BaseModel):
+class ValueUnitPair(BaseModel):
+    value: float
+    units: str
+
+    def __str__(self):
+        return f"{self.value} ({self.units})"
```

Comment:
```suggestion
        return f"{self.value} {self.units}"
```

Reply:
I just removed the brackets around the unit symbol. You can accept the suggestion straightaway on github I think